docs/transparent-validation/protectli/laboratory-assembly-guide.md

```diff
@@ -0,0 +1,436 @@
+# Laboratory stand dedicated to Protectli platforms assembly guide
+
+## Introduction
+
+This document describes the assembly procedure dedicated to the Protectli
+platforms (VP2410, VP2420, VP4630/VP4650/VP4670, V1210/V1410/V1610) testing
+stand.
+
+## Prerequisites
+
+The below table contains information about all elements which are needed to
+create the testing stand.
+
+=== "VP2410"
+
+    * VP2410 platform
+    * [RTE v1.1.0](https://shop.3mdeb.com/shop/open-source-hardware/open-source-hardware-3mdeb/rte/)
```

Comment:
Most of that seems to be common between boards. How about we have some common part, and some board-specific requirements, if any?